janusgraph-dist/src/assembly/static/cassandra/conf/cassandra.yaml

```diff
@@ -425,28 +589,36 @@ auto_snapshot: true
 # tombstones seen in memory so we can return them to the coordinator, which
 # will use them to make sure other replicas also know about the deleted rows.
 # With workloads that generate a lot of tombstones, this can cause performance
-# problems and even exhaust the server heap.
-# (https://www.datastax.com/dev/blog/cassandra-anti-patterns-queues-and-queue-like-datasets)
+# problems and even exaust the server heap.
+# (http://www.datastax.com/dev/blog/cassandra-anti-patterns-queues-and-queue-like-datasets)
```

Comment:
https://www.datastax.com/blog/2013/04/cassandra-anti-patterns-queues-and-queue-datasets

Reply:
I just copied this file from the original distribution with some smaller changes from our side. I don't want to change issues which came from the original. This would make it hard to see difference, if we update cassandra.